### PR TITLE
Carry My Luggage (Part 2)

### PR DESCRIPTION
**Pull request by jws-1**

### CODE REVIEW

File: tasks/carry_my_luggage/src/carry_my_luggage/state_machine.py [hunk 1]
```diff
@@ -5,10 +5,23 @@
 
 import rospy
 from lasr_person_following.msg import FollowAction
+from sensor_msgs.msg import Image
 
 
 class CarryMyLuggage(smach.StateMachine):
 
+    class GetImage(smach.State):
```
Comment (m-barker) [inline tasks/carry_my_luggage/src/carry_my_luggage/state_machine.py:13]: Is there a reason this is here rather than using the existing get_image skill?

**Review by m-barker — CHANGES_REQUESTED** [review]
Great work! Just one small comment

File: tasks/carry_my_luggage/src/carry_my_luggage/state_machine.py (hunk 1 above)
Comment (jws-1) [reply tasks/carry_my_luggage/src/carry_my_luggage/state_machine.py:13]: Just forgot that it existed! Good spot

**Review by m-barker — APPROVED** [review]
Nice! Great work!

**Pull request merged**